ui/src/components/CheckBoxSelectPair.vue

```diff
@@ -97,12 +96,21 @@ export default {
   data () {
     return {
       checked: false,
-      selectedOption: null
+      selectedOption: null,
+      selectOptionsTimer: null
     }
   },
   created () {
     this.checked = this.defaultCheckBoxValue
   },
+  watch: {
+    selectOptions () {
+      clearTimeout(this.selectOptionsTimer)
+      this.selectOptionsTimer = setTimeout(() => {
+        this.handleSelectOptionsUpdated()
```

Comment:
@shwstppr In my opinion, instead of changing the choice of options according to VPC, we can reset those options to null/blank to avoid repetition causing form crashes. What do you think?